config/default.yml

```diff
@@ -695,6 +694,11 @@ Rails/TimeZone:
     - strict
     - flexible
 
+Rails/TransactionRequiresNew:
+  Description: 'When using a transaction requires that the argument `requires_new: true` is passed.'
+  Enabled: true
```

Comment:
As far as I know, it's not a common rule. I think it should be `Enable: false` by default.